### DISCUSSION TITLE
SU2 V8 missing functions in PythonWrapper

### DISCUSSION THREAD

**Discussion by Igarizza** [su2code/SU2#2264] [Q&A] [answered]
Dear SU2 community,
in the v7.5 there were several useful functions in the pysu2 which I can't find anymore in V8:

- Get_DragCoeff
- Get_LiftCoeff
- Set/GetTemperature - now it is only  SetMarkerCustomTemperature
Is there any reason why these functions are removed or there is another way to get Lift and Drag coeff  / Get Temperatures in python? 

Thank you for your support!

**Comment by pcarruscag** [comment]
You can now access any output, please look at the examples in TestCases/pywrapper

**Reply by Igarizza** [reply]
Hi @pcarruscag. Thank you for your fast answer. I have checked the pywrapper TesCases. As I understand you use driver.GetOutputValue("VALUE") to get interested value from solver. Where can I find a list of possible values for OutputValues() ??

**Reply by bigfooted** [reply, marked as answer]
I think it can be any history output, and you can get that list of options by running SU2_CFD -d configfile.cfg
